lib/utils/get-element-type.js

```diff
@@ -1,5 +1,15 @@
 const {elementType, getProp, getPropValue} = require('jsx-ast-utils')
 
+function getConfigurationDefaultIfExists(settings, rawElement) {
+  // if a component configuration does not exists, return the raw element
+  if (!settings?.github?.components?.[rawElement]) return rawElement
```

Comment:
hmmmm, 🤔 lets ship it like this since its still 'experimental' and then decide if we want to add the boolean based on the results (i.e. testing it in product for a little bit) -- that will give us time to test if we need to add any variables to the `components` object.